srcpkgs/qmidiarp/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'qmidiarp'
+pkgname=qmidiarp
+version=0.6.5
```

Comment:
This release is from 2017... Things using Qt sometimes start accumulating more and more patches :/

Reply:
Hm, IME it's not worse than with other libs. Of course, there will likely have to be a patch for Qt6 at some point.